src/timeMachine/components/TagSelector.tsx

```diff
@@ -232,13 +266,38 @@ class TagSelector extends PureComponent<Props> {
       )
     }
 
+    // underscore values are not allowed for user data
+    // so there shouldn't be collisions
+    const selectedAll =
+      selectedValues.length === 1 && selectedValues[0] === '_all'
+
     return (
       <SelectorList
-        items={values}
+        items={values.filter(t => t !== '_all')}
         selectedItems={selectedValues}
         onSelectItem={this.handleSelectValue}
-        multiSelect={!this.props.isInCheckOverlay || this.isCompliant}
-      />
+        multiSelect={
+          !this.props.isInCheckOverlay ||
+          (this.isCompliant && isFlagEnabled('measurementMultiselect'))
+        }
+        disabled={selectedAll}
+      >
+        {this.isCompliant && isFlagEnabled('measurementMultiselect') && (
+          <List.Item
+            className="selector-list--item select-all"
+            testID="selector-list --select-all"
```

Comment:
```suggestion
            testID="selector-list--item select-all"
```
Is it missing `item`?

Reply:
oh no i wanted to keep this separate from the user generated list items for tests that are like "push 10 items in, expect there to be 10 selector-list--item test ids"